iclr2017/python/get-all-paper-status.py

```diff
@@ -0,0 +1,159 @@
+import openreview
+import requests
+import sys, getopt
+
+class PaperStatus:
+    Unassigned, Assigned, Commented, Reviewed, FullyReviewed = range(5)
+
+
+PaperStatusString = ["Unassigned", "Assigned", "Commented", "Reviewed", "FullyReviewed"]
+
+
+# get the info from the review, return NA if not there
+def get_score(content_type):
+    string_var = note.content.get(content_type, "NA")
+    string_var = string_var.split(':')[0]
+    return string_var
+
+
+# output_type: 1 text
+#               2 cvs
+#               3 html
+output_type = 2
+file_name = "output.txt"
+# parse optional arguments
+try:
+  opts, args = getopt.getopt(sys.argv[1:],"ho:t:",["ofile=","type="])
+except getopt.GetoptError:
+  print 'temp.py -o <outputfile> -t <type> 1: text, 2: csv'
+  sys.exit(2)
+
+for opt, arg in opts:
+  if opt == '-h':
+    print 'temp.py -o <outputfile> -t <type>'
+    sys.exit()
+  elif opt in ("-t", "--type"):
+    try:
+        output_type = int(arg)
+    except ValueError:
+        print 'Type must be 1 (for text) or 2 (for csv).'
+        sys.exit()
+    if (output_type is not 1) and (output_type is not 2):
+        print 'Type must be 1 (for text) or 2 (for csv).'
+        sys.exit()
+  elif opt in ("-o", "--ofile"):
+    file_name = arg
+my_file = open(file_name, "w")
+
+
+client = openreview.Client(username='OpenReview.net', password='OpenReview_beta', baseurl='https://openreview.net')
```

Comment:
1. the baseurl for localhost should be http://localhost:3000, instead of https (only the live site uses https)

2. there should be a setting somewhere in git that can help you change the default account name/email... I can help you out with this later today if you'd like